scripts/console.ts

```diff
@@ -38,20 +39,38 @@ let deployments: Deployments = require(`../deployments.json`);
 // TODO: Hardware Wallet support:
 // https://docs.ethers.io/v5/api/other/hardware/
 
-const CLONE_USAGE = `USER - onboard a project to GitConsensus (create a Token & Governor)`;
-const DEV_USAGE = `DEV - deploy GitConsensus, TokenFactory, or GovernorFactory`;
+const CONTRIBUTOR_USAGE = `CONTRIBUTOR - add or check a git commit/tag on-chain, Token interactions, or Governor interactions.`;
+const MAINTAINER_USAGE = `MAINTAINER - onboard a project to Git Consensus Protocol.`;
+const DEV_USAGE = `DEV - re-deploy GitConsensus, TokenFactory, or GovernorFactory contracts.`;
+
+const CONTRIBUTOR_ACTION_COMMIT_CHECK = `check a git commit`;
+const CONTRIBUTOR_ACTION_COMMIT_ADD = `add a git commit`;
+const CONTRIBUTOR_ACTION_TAG_CHECK = `check a git tag (verify a release)`;
+const CONTRIBUTOR_ACTION_TOKEN_CHECK = `check voting power (token balance)`;
+const CONTRIBUTOR_ACTION_TOKEN_DELEGATE = `delegate voting power to another address`;
+const CONTRIBUTOR_ACTION_PROPOSE = `create a proposal (for a release)`;
+const CONTRIBUTOR_ACTION_VOTE = `vote on an active proposal`;
+const CONTRIBUTOR_ACTION_EXECUTE = `execute an successful proposal (to add a release)`;
+
+const BOTH_CLONE = `Token and Governor`;
+const TOKEN_CLONE = `Token only`;
+const GOVERNOR_CLONE = `Governor only`;
 
-const BOTH_CLONE = `Token & Governor`;
-const TOKEN_CLONE = `Token`;
-const GOVERNOR_CLONE = `Governor`;
 const JSON_NUM_SPACES = 4;
 
 async function main(signer?: SignerWithAddress): Promise<void> {
     if (signer == undefined) {
         signer = await askForSigner();
     }
     switch (askForUsage()) {
-        case CLONE_USAGE:
+        case CONTRIBUTOR_USAGE:
```

Comment:
It might be better to factor our the nested `switch`es into their own functions for readability.

Reply:
Good idea! Though I will add TODO and punt this to address in a separate change where I can add functionality for each action.